src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -138,17 +134,33 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .WithDescription("A space seperated list of functions to load.")
                 .Callback(f => EnabledFunctions = f);
 
-            return base.ParseArgs(args);
+            Parser
+                .Setup<bool>("verbose")
+                .WithDescription("When false, hides system logs other than warnings and errors.")
+                .SetDefault(false)
```

Comment:
If we remove the default value here, would we still need `verboseLoggingArgExists`? We can probably just check if `VerboseLogging` is `null` or not.

Reply:
Parser is not binding properly to type `bool?` . I tried updating the nuget package same behavior.